src/Implementation/Exceptions/UnsupportedArchitectureException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Symbolica.Expression;
+
+namespace Symbolica.Implementation.Exceptions
+{
+    [Serializable]
+    public class UnsupportedArchitectureException : SymbolicaException
```

Comment:
Is this not one we want to show to users?

Reply:
Yes, and it was, but now more explicit since it is an `ErrorException`.